session/bootstrap.go

```diff
@@ -618,18 +618,34 @@ const (
 		INDEX time_index(time) COMMENT "accelerate the speed when querying with active watch"
 	) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_bin;`
 
-	// CreateRunawayQuarantineWatchTable stores the condition which is used to check whether query should be quarantined.
-	CreateRunawayQuarantineWatchTable = `CREATE TABLE IF NOT EXISTS mysql.tidb_runaway_quarantined_watch (
+	// CreateRunawayWatchTable stores the condition which is used to check whether query should be quarantined.
+	CreateRunawayWatchTable = `CREATE TABLE IF NOT EXISTS mysql.tidb_runaway_watch (
+		id BIGINT(20) NOT NULL AUTO_INCREMENT PRIMARY KEY,
 		resource_group_name varchar(32) not null,
-		start_time TIMESTAMP NOT NULL,
-		end_time TIMESTAMP NOT NULL,
-		watch varchar(12) NOT NULL,
+		start_time datetime(6) NOT NULL,
```

Comment:
How to deal with the timezone difference if the cluster is cross region deployed?

Reply:
`types.Time` always saves UTC time, so I think no problem.

Reply:
I don't see how the `Time` type will help you handle the timezone offset

Reply:
Now I convert all time to UTC and save, read and parse them as UTC

Reply:
So what is the benefit for this change, I didn't see any other table to use datetime to represent time

Reply:
Because timestamp only store time between 1970-01-01 00:00:01 UTC and 2038-01-19 03:14:07 UTC. You can find datetime in stats_history.